recipe/meta.yaml

```diff
@@ -33,6 +33,13 @@ requirements:
 test:
   imports:
     - pygeos
+  source_files:
+    - pygeos/test
```

Comment:
The error is `Did not find any source_files for test with pattern pygeos/test`. Does that not work for subdirectories?

(cc @ocefpaf)

Reply:
The problem here is that pygeos does not package the test directory.

I think this requires a download from GitHub to fix this.

Reply:
As far as I understand, that's exactly the goal of `source_files`, to ensure to copy files from the package source, if it's not included in the installed package

Reply:
See https://conda-forge.org/docs/maintainer/adding_pkgs.html#copying-test-files

Reply:
@ocefpaf do you have any idea here about the source_files ?

Reply:
It should work with subdirectories. Checking it now...

Reply:
@jorisvandenbossche the tests files are not there. source_files copy from source like @caspervdw mentioned above. We can either use a GitHub tarball or ship the tests with the PyPI sdist. (One could argue that we should always ship tests with in compiled extensions.) Or, the shortest route, copy the test files here. However, that last one is not sustainable in the long run b/c we need to keep them in sync with upstream.

Reply:
Ah, OK (will see if I can do a doc clarification PR then). 
I think long term we should maybe include the tests in the sdist then (but not in the wheel)?

Reply:
I would not add it in the wheel.